src/devtools/client/debugger/src/actions/types/ASTAction.js

```diff
@@ -1,5 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
-
-//
```

Comment:
Any reason why you're deleting this? I think we should keep the license for code that comes from moz. New files do not need it though.

Reply:
Oh yeah I meant to comment on this. All of these files were empty except for the license, so I deleted them, but lmk if they are actually serving a purpose and I can restore them!